docs/LibcxxSupport.md

```diff
@@ -1,63 +1,194 @@
+valarray | Yes | - |
+
+## Input/output Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+cstdio | Partial | All I/O functions implicitly call out to untrusted host. <br> **Supported functions:** snprintf(), sscanf(),  _vfscanf()*_, vsnprintf(), vsscanf(), sprintf(), vsprintf(), puts(), putchar(), vprintf(), printf(), _fprintf()*_, _getc()*_, _ungetc()*_, _fwrite()*_, _fflush()*_, _fputs()*_, _fputc()*_. <br> _* Only has support for the streams stderr and stdout, and does not set ferror_. | |
+fstream | No | - |
+iomanip | Partial | **Unsupported functions:** get_money(), put_money(), get_time(), put_time() |
+ios | Yes | Only basic support for C/POSIX locale. |
+iosfwd | Partial | See other headers for which forward declarations are supported. |
+iostream | Yes | - |
+istream | Yes | - |
+ostream | Yes | - |
+strstream | No | Header is provided, but deprecated as of C++98. |
 sstream | Yes | - |
-stddef | Yes |  - |
 streambuf | Yes | - |
-system_error | Yes | - |
+syncstream | No | Header is not provided, C++20 is not yet supported. |
+
+## Localization Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+clocale | Partial | Only basic support for C/POSIX locale. |
+codecvt | Partial |  Only basic support for C locale. Supported as part of C++11. |
+locale | Partial | Only basic support for C locale. |
+
+## Regular Expressions Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+regex | No | - |
+
+## Atomic Operations Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+atomic | Yes | Supported as part of C++11. |
+
+## Thread Support Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+condition_variable | Partial | Supported as part of C++11. Synchronization primitives are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. |
+future | Partial | Supported as part of C++11. Asynchronous invocations are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. |
+mutex | Partial | Supported as part of C++11. Synchronization primitives are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. <br> **Unsupported classes:** timed_mutex, recursive_timed_mutex. |
+shared_mutex | No | - |
 thread | No | - |
-tuple | Partial | **Supported function:** tie() |
-typeindex | Yes | - |
-typeinfo | No | - |
-type_traits | No | - |
-unordered_map | Yes | - |
-unordered_set | Yes | - |
-utility | Partial | **Unsupported function:** make_pair() |
-vector | Yes | - |
-version | No | - |
+
+## Filesystem Library
+Header | Supported | Comments |
+:---:|:---:|:---|
+filesystem | No | - |
+
+## C Compatibility Headers
+Header | Provided | Comments |
+:---:|:---:|:---|
+assert.h | No | - |
```

Comment:
Why is assert.h not supported?

Reply:
It's not provided by LLVM, the header doesn't exist if you only path the libcxx includes.